### ISSUE TITLE
CRX extension file as release asset

### ISSUE THREAD

**Issue by hmlendea** [tryton-vanmeer/ProtonDB-for-Steam#15]
Hello,

I see that the Chrome version did exist in the past, but it was taken down.
This is very unfortunate (read: stupid on Google's part).

Could you, though, do build the Chrome version, and provide the CRX file so that we can install it? You can upload it as a Release asset here on GitHub.

We'd have to manually install it, which is pretty uncomfortable, but at least it's something.

Thanks.

**Comment by tryton-vanmeer** [comment]
I figured it would be a tedious option to sideload extensions in Chrome, but if people want it as an option, I'm willing to include a CRX file as part of the releases.

Of course I decided to test in Chromium first before building and ran into a issue. Some restrictions were made with how cross-origin requests are made with extensions (Since Chrome 73).

Some links for further reading:
+ [Changes to Cross-Origin Requests in Chrome Extension Content Scripts](https://www.chromium.org/Home/chromium-security/extension-content-script-fetches)
+ [Manage Events with Background Scripts](https://developer.chrome.com/extensions/background_pages)

I'll be making an update to the extension and the new release will include a CRX file.
